workshops/3D Heatmap/README.md

```diff
@@ -0,0 +1,204 @@
+
+## Setting up
+
+- Repl is an online, instant development environment to learn, build, collaborate, and host your project. Show you don’t have to waste time while setting up a development environment. Creating an account on repl is super easy.
+
+[Create a repl](https://repl.it/signup)
+
+
+![repl website](https://cloud-73h0sldam.vercel.app/0screenshot_2020-12-25_at_23.03.53.png)
+
+- Create a repl by clicking **+ New repl** and choose programming language **PYTHON** and named it as you want and **create repl**
+
+- Download the dataset from below, drag and drop them to the folder you are working on repl.\
+[volcano.csv](https://cloud-8ycpvzexa.vercel.app/0volcano.csv)
+
+
+- Rename the file as volcano.csv and place it in the repl folder in which you are working.
+
+- The setup is complete. ✅.
+
+**Advantage of working on repl**
+
+*When we program in repl, it automatically imports libraries and dependencies with a simple command. So, you just require a repl account or any python idle get started.*
+
+![Library](https://cloud-1th3ydnib.vercel.app/1workshop_library.gif)
+
+## Pandas🐼
+Pandas is a popular Python-based toolkit. It presents a diverse range of utilities like converting an entire data table into a NumPy matrix array and much more. That makes pandas a trusted ally in data science and machine learning.
+
+## Numpy🔢
+We use NumPy to work with arrays to perform a large set of mathematical operations. An array is a data structure consisting of a collection of elements, each identified by at least one array index or key.
+
+
+## Matplotlib📊
+A picture is worth a thousand words, and with Python’s matplotlib library, fortunately, it takes far less than a thousand words of code to create a production-quality graphic.
+
+## Toolkits🧰
+Toolkits are collections of application-specific functions that extend Matplotlib.
+
+## Seaborn䷀
+Seaborn library is one of the rarest earth metals. Seaborn, which provides a variety of visualization patterns, uses fewer syntax and decent themes.
+Great Right!😲
+
+# 1. Making 3D Plane
+
+
+Importing libraries
+```python
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+```
+The first step is to create 3D axes and then plot any of the 3D graphs. We pass ```projection='3d'``` to plt.axes, This is the empty canvas that we'll be painting.
+
+```python
+fig = plt.figure()
+ax = plt.axes(projection='3d')
+plt.show()
+```
+![3D Plane](https://cloud-a08hob7s0.vercel.app/1workshop2.gif)
+
+# 2. Plotting Graph
+
+Importing Libraries
+```python
+from mpl_toolkits.mplot3d import Axes3D
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+import numpy as np
+```
+The most basic three-dimensional plot is a line or collection of scatter plot created from sets of (x, y, z) triples.
+
+```
+ax = plt.axes(projection='3d')
+z = np.linspace(0, 30, 100)
+```
+Initialise x is array of np.sin(z), and similarly y is a array of np.cos(z)
```

Comment:
What does this mean?

Reply:
> What does this mean?
z is the function of x and y which is stored in the array.